### ISSUE TITLE
VueJS support

### ISSUE THREAD

**Issue by supapo** [InseeFrLab/keycloakify#3]
This is awesome project,  thank you. 
Can it support other technologies like VueJS? 

I saw that the base Keycloak theme is generating JSON files based on the realm configuration
What else do we need?

**Comment by garronej** [comment]
Hi,  
As of right now, it only supports React. To support Vue at least [these](https://github.com/InseeFrLab/keycloakify/tree/main/src/lib/components) components would have to be re-wrote.  

I don’t know exactly what you mean by "the base Keycloak theme is generating JSON" could you point me to the file you are referring to?

**Comment by supapo** [comment]
Thank you, @garronej.
I meant this folder src/bin/build-keycloak-theme/generateFtl
As I can see, you took the flags from Keycloak's template into JS files.

**Comment by garronej** [comment]
Yeah... I am not particularly proud of the way I implemented that.  
I should not have a specific .ftl to js script for every page but rather use something like: 
```ftl
<#list .data_model?keys as var>
    ${var}
</#list>
```
I will re-write it when I got time.   

As for Vue.js support... the implementation is very dependent on how `create-react-app` bundle the app.
The input data keycloakify works with is the output of `yarn build`.
